ts/util/requests.ts

```diff
@@ -15,6 +15,16 @@ export function post(url: string, body: any) {
   });
 }
 
+export function get(base: string, body: string) {
```

Comment:
rename "base" -> "url" since this is the full URL. "Base URL" usually means just the server name